### DISCUSSION TITLE
Adding new image shuffling method for Dreambooth?

### DISCUSSION THREAD

**Discussion by samf1986** [TheLastBen/fast-stable-diffusion#334] [Ideas]
Any chance you'll add this method?

https://www.reddit.com/r/StableDiffusion/comments/yj588o/improve_your_dreambooth_trainings_heres_the_source/

"Shivam‘s implementation shuffles the list of instance images when the dataset is created. The dataloader then shuffles the created pairs but the pairs remain the same throughout the training.

With my change the dataset is recreated before each epoch starting with the second epoch. Furthermore the pairs in the dataset are created so that every image is randomly selected for at least one pair."

**Comment by TheLastBen** [comment]
Have you noticed any improvement in quality with this feature ?
